Add support for loud comments

simoneldevig opened the pull request:
It is very important the add support for loud comments when working in production environments since normal comments are removed when compiling sass with as compressed.

7rulnik commented:
@simoneldevig hi! Thank you for PR.
I think we can use `String.prototype.includes` instead of two equality checks. Can you do it?

simoneldevig commented:
@7rulnik I've tried to change it to use includes, but since the text that does not contain flexibility-disable is undefined it means that it would require an ekstra check anyways? :-)

7rulnik commented:
@simoneldevig you can use default argument `rule.some(({prop, text = ''}) =>`

simoneldevig commented:
@7rulnik I've changed the check to use endsWith since I ran in some issues with includes on not loud comments. I also added a test for loud comments as well :-)

7rulnik commented:
@simoneldevig nice! Thank you for tests. I will merge and release it at saturday.

7rulnik commented:
@simoneldevig just released it in [v.1.0.1](https://github.com/7rulnik/postcss-flexibility/releases/tag/v1.1.1). Thanks!
